lib/helper/REST.js

```diff
@@ -219,5 +219,175 @@ class REST extends Helper {
 
     return this._executeRequest(request);
   }
+
+
+  /**
+ * Checks if the response code is equal to what was provided
+ *
+ * ```js
+ * I.seeResponseCodeIs(200);
+ * ```
+ *
+ * @param {string|number} statusCode
+ */
+  seeResponseCodeIs(statusCode) {
+    expect(this.response.status).to.equal(statusCode, `Status code ${this.response.status} of response is not equal to ${statusCode}`);
+  }
+
+  /**
+   * Checks if the response code is not equal to what was provided
+   *
+   * ```js
+   * I.dontSeeResponseCodeIs(200);
+   * ```
+   *
+   * @param {string|number} statusCode
+   */
+  dontSeeResponseCodeIs(statusCode) {
+    expect(this.response.status).to.not.equal(statusCode, `Status code ${this.response.status} of response is equal to ${statusCode}`);
+  }
+
+  /**
+   * Checks if the response code is an information response code by checking if its value is between 100-199
+   *
+   * ```js
+   * I.seeResponseWasInformation();
+   * ```
+   */
+  seeResponseWasInformation() {
+    expect(this.response.status).to.be.above(99);
+    expect(this.response.status).to.be.below(200);
```

Comment:
I would think instead of doing this, we can do something like:

- seeResponseWasInformation(from, to): if the range is provided, the assertion will use that range.
- otherwise, using the default (between 100-199)

Reply:
I think I.seeResponseCodeInRange(from,to) would be more expressive

Reply:
The range for informational messages is set at 100-199, if you want to assert on a specific http status code you wouldn't be using this method. Providing a range is pointless.

Imagine you have an API where you only want to assert you got a good response back. This would mean you are wanting to check the status code is successful. The successful range is 200-299. In your approach it would be

```js
I.sendPostRequest('/url')
I.seeResponseCodeInRange(200-206)
```
Not very readable. A new dev will ask, "What are the meanings behind these magic numbers? Why is 200-206 ok but 207+ not ok? What is going on here???" A couple google searches will show that 200 status codes are successful status codes and that there are only 6 of them. Now look at the readability of the test when you use what has been implemented
```js
I.sendPostRequest('url')
I.seeResponseWasSuccessful()
```

500x more readable

Reply:
I would say adding a new method to assert on a range rather than editing these generic methods. Looking at our companies end to end tests these range methods would never be used. They would actually be pointless. You usually want to assert on a specific status code or generically ensure you got a good response. No one cares it falls between some specific range of magic numbers but instead that you got a good, bad, or whatever response

Reply:
You also would never assert on a range crossing into other status codes. There is not a single use case I can think of that would be something like
```js
I.seeResponseCodeInRange(105, 351)
```

When would that ever be useful?

Reply:
@PeterNgTr Added the response between function as requested so you can use that instead of providing a range to the success, client error, and server error response code methods.